test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -101,6 +100,75 @@ Describe "TabCompletion" -Tags CI {
 
     It 'Should complete keyword' -skip {
         $res = TabExpansion2 -inputScript 'using nam' -cursorColumn 'using nam'.Length
-        $res.CompletionMatches[0].CompletionText | Should be 'namespace'
+        $res.CompletionMatches[0].CompletionText | Should Be 'namespace'
+    }
+
+    Context NativeCommand {
+        BeforeAll {
+            $nativeCommand = (Get-Command -CommandType Application -TotalCount 1).Name
```

Comment:
It might be wiser to create an external command on testdrive and change the path, but that's a tiny bit annoying to make it portable, so I'll merge as is.